examples/phaser/ADAR_pyadi_functions.py

```diff
@@ -0,0 +1,164 @@
+# ADAR_functions.py
+
+import pickle
+
+import numpy as np
+
+# MWT: probably better to pass verbosity to each function
+verbose = True
+
+
+def ADAR_init(beam):
+    # Initialize the ADAR1000
+    beam.reset()  # Performs a soft reset of the device (writes 0x81 to reg 0x00)
+    beam._ctrl.reg_write(
+        0x400, 0x55
+    )  # This trims the LDO value to approx. 1.8V (to the center of its range)
```

Comment:
Is there a property for this feature?

Reply:
Still investigating this. The driver sets the LDO trim on probe, but the reset function may clobber it. (The reset function may not be needed.)

Reply:
Yup, not necessary. This exact operation is done by the driver on probe:
https://github.com/analogdevicesinc/linux/blob/master/drivers/iio/beamformer/adar1000.c#L2552
The beam.reset() would have clobbered it, but the reset is also done by the driver, so took that out, too.